packages/forms-web-app/src/controllers/examination/middleware/submission.middleware.js

```diff
@@ -0,0 +1,31 @@
+const {
+	getExaminationSubmissionComplete,
+	getExaminationUploadingState
+} = require('../session/examination-session');
+
+const {
+	routesConfig: {
+		examination: {
+			directory,
+			pages: { submissionComplete }
+		}
+	}
+} = require('../../../routes/config');
+
+const isProcessingSubmission = (req, res, next) => {
+	const { url, session } = req;
+
+	if (url === submissionComplete.route) return next();
```

Comment:
would be nice not have this. But seems required to lock all the other routes out and allow the submission complete to be shown. 

We can flip the logic below so if no condition next etc.